### ISSUE TITLE
No plugin found for a rule: "apply-numeric-separators" (putout)

### ISSUE THREAD

**Issue by gnarea** [EvgenyOrekhov/eslint-config-hardcore#655]
I get the error above with a fresh install. See:

https://github.com/EvgenyOrekhov/eslint-config-hardcore/blob/9167131fd617067b5678fa13b7f998dde4ba2d5a/base.json#L793

Seems to be fixed in #643, but that PR requires renaming the rules, I think.

**Comment by gnarea** [comment]
I tried pinning the putout dependencies to the ones in your `package.json`:

```shell
 npm install -D eslint-plugin-json eslint-plugin-putout@^13 putout@^24 @putout/plugin-apply-shorthand-properties@^3
```

But then I get an error when running `eslint`:

```
Oops! Something went wrong! :(

ESLint: 8.28.0

TypeError: Error while loading rule 'putout/putout': Cannot read properties of undefined (reading 'typeAnnotation')
Occurred while linting /home/gus/repos/dnssec-verifier-js/src/lib/UnverifiedChain.ts
    at FastPath.call (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/fast-path.js:123:22)
    at genericPrintNoParens (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:1583:22)
    at genericPrint (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:123:30)
    at print (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:77:15)
    at /home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:49:41
    at printComments (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/comments.js:281:22)
    at print (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:55:49)
    at /home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:49:41
    at FastPath.call (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/fast-path.js:126:18)
    at genericPrintNoParens (/home/gus/repos/dnssec-verifier-js/node_modules/@putout/recast/lib/printer.js:1741:22)

```

**Comment by EvgenyOrekhov** [comment]
I'm planning to upgrade Putout to the latest version (it will be a huge upgrade).

**Comment by EvgenyOrekhov** [comment]
Should be fixed in [v26.0.0](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v26.0.0).
